tracers/logger/logger_test.go

```diff
@@ -105,3 +116,111 @@ func TestStructLogMarshalingOmitEmpty(t *testing.T) {
 		})
 	}
 }
+
+func TestFormatLogs(t *testing.T) {
+	logs := []StructLog{
+		{Pc: 1, Op: vm.PUSH1, Gas: 100, GasCost: 2, Depth: 1, Memory: []byte("test"), Stack: []uint256.Int{*uint256.NewInt(1)}},
+	}
+
+	formattedLogs := formatLogs(logs)
+	if len(formattedLogs) != len(logs) {
+		t.Errorf("Expected %d formatted logs, got %d", len(logs), len(formattedLogs))
+	}
+}
+
+func TestCaptureStart(t *testing.T) {
+	logger, _ := NewStructLogger(nil)
+	env := &vm.EVM{}
+
+	logger.CaptureStart(env, common.Address{}, common.Address{}, false, nil, 0, nil)
+	logger.CaptureEnd(nil, 1234, fmt.Errorf("Some Error"))
+	logger.CaptureClauseEnd(10000)
+	logger.Stop(fmt.Errorf("Some Error"))
+	logger.CaptureClauseStart(1234)
+	logger.CaptureClauseEnd(1234)
+	logger.Reset()
+}
+
+func TestNewMarkdownLogger(t *testing.T) {
+	writer := &bytes.Buffer{}
+	cfg := &Config{EnableMemory: true}
+	logger := NewMarkdownLogger(cfg, writer)
+
+	if logger.cfg != cfg {
+		t.Errorf("Expected cfg to be set correctly")
+	}
+
+	env := &vm.EVM{}
+
+	logger.CaptureStart(env, common.Address{}, common.Address{}, false, nil, 0, nil)
+	logger.CaptureEnd(nil, 1234, fmt.Errorf("Some Error"))
+	logger.CaptureClauseEnd(10000)
+	logger.CaptureClauseStart(1234)
+	logger.CaptureClauseEnd(1234)
+}
+
+func TestWriteLogs(t *testing.T) {
+	writer := &bytes.Buffer{}
+
+	logs := []*types.Log{
+		{
+			Address:     common.HexToAddress("0x1"),
+			Topics:      []common.Hash{mockHash("topic1"), mockHash("topic2")},
```

Comment:
`common.BytesToHash([]byte("topic1"))`

Reply:
Can you elaborate a bit on this one?

Reply:
The function `mockHash` can be replaced by common.BytesToHash

Reply:
Let me know if this is what you wanted